src/phoenix/db/helpers.py

```diff
@@ -328,3 +328,30 @@ def _date_trunc_for_sqlite(
 
     # Convert back to UTC by subtracting the offset
     return func.datetime(t, f"{-utc_offset_minutes} minutes")
+
+
+def get_ancestor_span_ids(parent_id: str) -> Select[tuple[int]]:
```

Comment:
Due to the ambiguity with the `str` span_id, the existing convention is to use `rowid` instead
```suggestion
def get_ancestor_span_rowids(parent_id: str) -> Select[tuple[int]]:
```